### DISCUSSION TITLE
Where is an apk for download?

### DISCUSSION THREAD

**Discussion by CleanFuels** [j-fbriere/squawker#248] [General]
Where is an apk for download? F-droid does not present the latest release

**Comment by j-fbriere** [comment]
There are two ways to get the latest version rapidly:

1- The [project's main page](https://github.com/j-fbriere/squawker) specify the URL of an alternate F-Droid type repository (that I host):
[https://apkrep.creativaxion.org/fdroid/repo?fingerprint=443DA0A316DFB86BFD05D0123951855E7CD8724969FAD66D6E62EB801299744A](https://apkrep.creativaxion.org/fdroid/repo?fingerprint=443DA0A316DFB86BFD05D0123951855E7CD8724969FAD66D6E62EB801299744A)
Add this repository in your F-Droid client app to get the latest version.

2- Download the app from the [project's latest version on github](https://github.com/j-fbriere/squawker/releases/tag/v3.7.7) (look at the bottom of the page.

**Important Note** 
In case Squawker is already installed on your device and you want to reinstall it or want to install a version from another repository (from F-Droid to github or from F-Droid to the alternate F-Droid repository for instance), make sure to backup your application data (Settings/Data, tap Export, select all items then tap the save icon) and uninstall Squawker before proceeding. After you have reinstalled Squawker from the new repository, import your backup (Settings/Data, tap Import).
